test/README.md

```diff
@@ -138,14 +138,14 @@ environment.
        helm \
        kubectl \
        kustomize \
+       qemu \
        lima \
        minio-mc \
        velero \
        virtctl
    ```
 
-   lima version 1.0.0 or later is required, latest version is
-   recommended.
+   Tested with lima version 1.0.4.
```

Comment:
We still require lima version 1.0.0 or later, and we still recommend to use the latest version. We need need to add a note about the tested version.